lib/calendar.js

```diff
@@ -58,15 +61,27 @@ export default class Calendar {
         csreq=0
       WHERE client='?' AND service='kTCCServiceCalendar'`.replace(/\n/g, ' ');
     } else {
-      query = this.xcodeVersion.major < 10 ?
-        `INSERT INTO 'access' VALUES ('kTCCServiceCalendar', '?', 0, 1, 1, 0, 0);` :
-        `INSERT INTO 'access' VALUES ('kTCCServiceCalendar', '?', 0, 1, 1, 0, 0, 0, 0, 0, 0);`;
+      let count = this.columnCount;
+      if (_.isEmpty(this.columnCount)) {
```

Comment:
note, that isEmpty returns true for primitive types, like numbers

Reply:
That was the intention. Though I guess `_.isNumber` would better signal the intent.